fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ImmutableAvailability.java

```diff
@@ -97,11 +101,15 @@ public ImmutableAvailability(
                 entry -> new SimplifiedIntervalList(entry.getValue());
 
         Map<Column, List<Interval>> map = dimensionIntervals.entrySet().stream()
+                .filter(entry -> dimensionDictionary.findByApiName(entry.getKey()) != null)
```

Comment:
Would it make sense to have a debug level log statement here that logs the fact that we're filtering out a dimension value? I could see people getting confused if they are working on exposing a dimension across multiple services, and they accidentally skipped a step.